packages/odyssey-react-mui/src/useUniqueId.ts

```diff
@@ -14,7 +14,7 @@ import { useMemo } from "react";
 
 import { createUniqueId } from "./createUniqueId";
 
-export const useUniqueId = (id?: string): string => {
+export const useUniqueId = (id?: string) => {
   const uniqueId = useMemo(() => createUniqueId(), []);
 
   return id || uniqueId;
```

Comment:
Same as above.

Reply:
Yeah, this was existing code, but I can remove it.

Reply:
I can't find this in the current code.